addons/analytic/security/ir.model.access.csv

```diff
@@ -1,6 +1,7 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_account_analytic_account,access_account_analytic_account,model_account_analytic_account,group_analytic_accounting,1,1,1,1
 access_account_analytic_line,access_account_analytic_line,model_account_analytic_line,group_analytic_accounting,1,1,1,1
-access_account_analytic_tag,access_account_analytic_tag,model_account_analytic_tag,group_analytic_accounting,1,1,1,1
+access_account_analytic_tag,access_account_analytic_tag,model_account_analytic_tag,group_analytic_tags,1,1,1,1
+access_account_analytic_tag_user,access_account_analytic_tag_user,model_account_analytic_tag,base.group_user,1,0,0,0
```

Comment:
You don't really `hide that record if user have no access to tag.` so your commit message is not really right. 🤔